README.md

```
@@ -1,116 +1,136 @@
-<a href="https://github.com/RamakrushnaBiswal/PlayCafe/issues"><img src="https://img.shields.io/github/issues/RamakrushnaBiswal/PlayCafe?color=0059b3"></a>
+    <a href="https://github.com/RamakrushnaBiswal/PlayCafe/network/members"><img src="https://img.shields.io/github/forks/RamakrushnaBiswal/PlayCafe?color=yellow"></a>
+    <a href="https://github.com/RamakrushnaBiswal/PlayCafe/issues"><img src="https://img.shields.io/github/issues/RamakrushnaBiswal/PlayCafe?color=0059b3"></a>
+</p>
 
 <hr/>
 
+## 📚 Table of Contents
+1. [✨ Project Overview](#-project-overview)
+2. [🌐 Demo](#-demo)
+3. [🔥 Features](#-features)
+4. [🛠️ Tech Stack](#-tech-stack)
```

Markdownlint

Link fragments should be valid

(MD051, link-fragments)